### ISSUE TITLE
[Request]Resize Modules

### ISSUE THREAD

**Issue by C0nvert** [edrys-org/edrys#15]
Hi is there a possibility to resize Modules?

**Comment by andre-dietrich** [comment]
At the moment not, there is a possibility that is only exposed when you export your Lab-Configuration to JSON, there you will see a size option, but this is currently not activated. Thus, all will have the same size ...

**Comment by edrysorg** [comment]
That is true. Maybe we can use this issue as an opportunity to discuss the best way to lay out modules in the future?
